### PR TITLE
feat(sumologic-kubernetes-setup): add Dockerfile

### PR DESCRIPTION
**Pull request by pmalek-sumo**

### CODE REVIEW

File: sumologic-kubernetes-setup/v3.0.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
+FROM registry.access.redhat.com/ubi8/ubi:8.4
+
+### Required Labels #####
+LABEL name="SumoLogic Kubernetes Setup" \
+      vendor="SumoLogic" \
+      version="1.0" \
+      release="1" \
+      summary="SumoLogic k8s setup job image" \
+      description="This image is used to setup the kubernetes cluster and SumoLogic environment before installing SumoLogic collection helm chart"
```
Comment (sumo-drosiek) [inline sumologic-kubernetes-setup/v3.0.0/Dockerfile:9]: please add maintainer label

Comment (pmalek-sumo) [reply sumologic-kubernetes-setup/v3.0.0/Dockerfile]: Done

**Review by sumo-drosiek — APPROVED** [review]

File: sumologic-kubernetes-setup/v3.0.0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,46 @@
+FROM registry.access.redhat.com/ubi8/ubi:8.4
+
+### Required Labels #####
+LABEL name="SumoLogic Kubernetes Setup" \
+      vendor="SumoLogic" \
+      version="1.0" \
```
Comment (kkujawa-sumo) [inline sumologic-kubernetes-setup/v3.0.0/Dockerfile:6]: We are setting version to the version of used component so here it should be `3.0.0`

**Review by kkujawa-sumo — APPROVED** [review]

File: sumologic-kubernetes-setup/v3.0.0/Dockerfile (hunk 2 above)
Comment (pmalek-sumo) [reply sumologic-kubernetes-setup/v3.0.0/Dockerfile]: Done, thanks!

**Pull request merged**